practice/src/css/layouts/container.css

```diff
@@ -14,7 +14,9 @@
 /* Desktop */
 @media (min-width: 1440px) {
   .container {
+    padding-left: 0px;
```

Comment:
Only use padding-left: 0; and padding-right: 0. Don't need to use px for 0 value